installer/functions/account/get-account-info.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+
+export const handler: ServerlessFunctionSignature = async function(
```

Comment:
a more idiomatic function expression would use arrow functions like:

```tsx
export const handler: ServerlessFunctionSignature = async (
  context: Context,
  event: {},
  callback: ServerlessCallback
) => {
  ...
}
```

also, is there any reason you're explicitly typing the function parameters? as far as i see, `ServerlessFunctionSignature` should already type these parameters, and allows you to further add template types if necessary like `ServerlessFunctionSignature<T, U>`:

![image](https://user-images.githubusercontent.com/6936736/155483309-5a5235dd-48a1-45bd-9726-1ddef2d80c33.png)

so instead of overriding the types, you could write this like:

```tsx

export const handler: ServerlessFunctionSignature = async (context, event, callback) => {
  ...
}
```

and the parameters should still be typed

![image](https://user-images.githubusercontent.com/6936736/155483759-269e042d-de0d-47c3-a45c-af86fb7a5782.png)